webapp/channels/src/components/suggestion/command_provider/app_provider.tsx

```diff
@@ -31,19 +31,11 @@ type Props = {
     rootId?: string;
 };
 
-export type Results = {
```

Comment:
A lot of these files had their own types for the results and the results callback, so I created unified ones in `provider.tsx`. Those ones are also generic so that we have better types for more of this code